src/main/java/dk/kb/present/View.java

```diff
@@ -159,14 +163,35 @@ public String apply(DsRecordDto record) {
      * @param record to extract basic metadata from.
      */
     private Map<String,String> createBasicMetadataMap(DsRecordDto record) {
+
         final Map<String, String> metadata = new HashMap<>();
         metadata.put("recordID", record.getId());
         metadata.put("origin", origin);
+        metadata.put("mTime", getSolrDate(record.getmTimeHuman()));
         //TODO: Update placeholder when actual value is in place
         metadata.put("conditionsOfAccess", "TODO: placeholderCondition");
         return metadata;
     }
 
+    /**
+     * Ensure formatting of date from ds-storage is indexable in solr and correctly formatted for schema.org representation
+     * by converting from format {@code yyyy-MM-dd HH:mm:ssZ} to {@code yyyy-MM-ddTHH:mm:ssZ}.
+     * @param dateTime a string representation of a dateTime in the format {@code yyyy-MM-dd HH:mm:ssZ}.
+     * @return a solr and schema.org compliant string in the format {@code yyyy-MM-ddTHH:mm:ssZ}
+     * converted with the {@link DateTimeFormatter#ISO_INSTANT}.
+     */
+    private static String getSolrDate(String dateTime) {
+        // Define the formatter for the input date string. This is almost already the expected format of ISO 8601.
+        // However, a small conversion needs to be make to change the space in the input pattern to a T.
+        DateTimeFormatter dsStoragePattern = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ssZ", Locale.ROOT);
+        OffsetDateTime offsetDateTime = OffsetDateTime.parse(Objects.requireNonNull(dateTime), dsStoragePattern);
+        Instant instant = offsetDateTime.toInstant();
+        // Convert the datetime string to the ISO INSTANT format, which is in use in both SCHEMA.ORG and SOLR
+        // https://schema.org/DateTime and https://solr.apache.org/guide/6_6/working-with-dates.html#WorkingwithDates-DateFormatting
+        DateTimeFormatter isoInstantFormatter = DateTimeFormatter.ISO_INSTANT;
```

Comment:
Solr supports millisecond granularity: https://solr.apache.org/guide/6_6/working-with-dates.html

I suggest we use the "non-human" `mTime` (which is microsecond granularity) as input instead of `mTimeHuman`.

Reply:
Addendum: We should consider if we want to keep the whole Epoch microsecond-thing in `ds-storage`. It is an uncommon measurement and maybe the price (a maximum of 1000 records ingested per second) for shifting to Epoch milliseconds is acceptible?

Reply:
The method getSolrDate isn't needed in the new implementation, however I haven't removed it because it will be needed again, when we are to include a last modified in origin date.